packages/vite/src/index.ts

```diff
@@ -171,7 +174,8 @@ export function imagetools(userOptions: Partial<VitePluginOptions> = {}): Plugin
           const fileHandle = this.emitFile({
             name: basename(pathname, extname(pathname)) + `.${metadata.format}`,
             source: image ? await image.toBuffer() : await readFile(`${cacheOptions.dir}/${id}`),
-            type: 'asset'
+            type: 'asset',
+            originalFilename: relative(rootPath, id)
```

Comment:
it looks like vite also normalizes the path, so we should probably do the same:

https://github.com/vitejs/vite/blob/31cc61b7404b2b22d956856f97eeebd0a62f3b4d/packages/vite/src/node/plugins/asset.ts#L397

Reply:
I guess you also need to add also `names: [basename(pathname, extname(pathname)) + `.${metadata.format}`]`, Vite 6 manifest using `asset.names.length > 0` to check if the asset with `type='assets'` should be added to the manifest:

![image](https://github.com/user-attachments/assets/7d403e77-c72f-4fcc-85c3-3805d836f0cf)


`name` was deprecated time ago, Vite<6 checking if `name` is present to include the asset in the manifets:
![image](https://github.com/user-attachments/assets/739c6927-984d-4704-a162-b9786163a876)